omero/sysadmins/omeroweb-upgrade.rst

```diff
@@ -87,6 +87,58 @@ All official OMERO.web plugins can be installed from PyPI_.
 You should remove all previously installed plugins and install the latest
 versions using pip.
 
+.. _clearing_session_store:
+
+Clear the sessions store (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During an upgrade, it might be necessary to clear the sessions store
+before restarting OMERO.web. This is the case when either OMERO.web or
+Django introduced backwards incompatible changes in the way sessions are
+stored or if the session serialization format has been modified via
+:property:`omero.web.session_serializer`.
+
+.. warning::
+
+   Clearing the sessions store will terminate all OMERO.web sessions and log
+   out all users.
+
+The process for clearing the session store depends on the storage backend:
+
+- OMERO.web session data is stored on the filesystem if the
```

Comment:
data is plural. We have a mix of "data is" and "data are" in our doc.
We will need to fix that

Reply:
Oh dear, I didn't mean to enter such an unchartered territory. To my defense, the Django documentation is extensively use of `session data` with the singular form of the verb. 😄 
I think I'll just use `sessions` instead of `session data` everywhere.